packages/code-studio/src/storage/LocalWorkspaceStorage.ts

```diff
@@ -72,8 +72,11 @@ export class LocalWorkspaceStorage implements WorkspaceStorage {
 
   private layoutStorage: LayoutStorage;
 
-  constructor(layoutStorage: LayoutStorage) {
+  private serverVersion?: string;
+
+  constructor(layoutStorage: LayoutStorage, serverVersion?: string) {
     this.layoutStorage = layoutStorage;
+    this.serverVersion = serverVersion;
```

Comment:
I don't think `LocalWorkspaceStorage` needs the `serverVersion`, revert the changes in this file.